articles/integrations/aws-api-gateway/part-3.md

```diff
@@ -135,7 +135,6 @@ Open the `index.html` file located in the root of your `pets` folder to add all
 <script type="text/javascript" src="lib/CryptoJS/rollups/hmac-sha256.js"></script>
 <script type="text/javascript" src="lib/CryptoJS/rollups/sha256.js"></script>
 <script type="text/javascript" src="lib/CryptoJS/components/hmac.js"></script>
-<script type="text/javascript" src="lib/CryptoJS/components/enc-base64.js"></script>
```

Comment:
right, but if we don't know why it was added, IMHO we shouldn't remove it. Unless you can try the code and ensure everything is ok without the library.